03-bash/task1.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+is_integer () {
+	local re='^[0-9]+$'
+	[[ $1 =~ $re ]]
+	return $?
+}
+
+integer_to_zero_string () {
+	local ret
+	printf -v ret "%04d" $1
+	echo "$ret"
+}
+
+is_ukr_locale() {
+	local re=$(locale | grep 'LANG=uk_UA')
+	if [ "$re" == '' ]; then
```

Comment:
-z is usual way to compare with empty line

Reply:
done